pd-control.md

```diff
@@ -765,7 +787,25 @@ Usage:
 - `src-tolerance-ratio` and `dst-tolerance-ratio` are configuration items for the expectation scheduler. The smaller the `tolerance-ratio`, the easier it is for scheduling. When redundant scheduling occurs, you can appropriately increase this value.
 
     ```bash
-    >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.05
+       >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
+    ```
+
+- `read-priorities`,`write-leader-priorities`, and `write-peer-priorities` control the first and second dimensions which have the first priority of balance when different types of hotspots are delt with. For hotspots of  `read` and `write-leader` types, the available dimensions are `query`, `byte`, and `key`. For hotspots of `write-peer`, the available dimensions are `byte` and `key`. If not all the cluster components are upgraded to v5.2 and later, these configurations will not take effect, and the compatible configurations will be used. Usually, you do not need to modify these configuration items.
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set read-priorities query,byte
+    ```
+
+- `strict-picking-store` is a switch that controls the search space of hotspot scheduling. When it is enabled and if stability is ensured, hotspot scheduling will start. Usually it is enabled. When it is disabled, only the balance of dimensions of the first priority is enusred, which might reduce the balance of other dimensions. Usually, you do not need to modify this configuration item.
```

Comment:
`When it is enabled and if stability is ensured, hotspot scheduling will start` confuse me